core/src/components/back-button/test/a11y/back-button.e2e.ts

```diff
@@ -1,6 +1,32 @@
+import AxeBuilder from '@axe-core/playwright';
 import { expect } from '@playwright/test';
 import { configs, test } from '@utils/test/playwright';
 
+/**
+ * Only ios mode uses ion-color() for the back button
+ */
+configs({ directions: ['ltr'], modes: ['ios'], themes: ['light', 'dark'] }).forEach(({ title, config }) => {
+  test.describe(title('back-button: contrast'), () => {
```

Comment:
```suggestion
  test.describe(title('back-button: a11y'), () => {
```

Let's keep it consistent with the other tests especially since since the test is checking for all violations. It's not specific to contrast. The same goes for the rest.

Reply:
I guess I could. My reasoning for changing it (although perhaps I should have changed it to something else) is that we aren't really testing the accessibility of these components very much when we aren't testing very many versions of them. For example, this test only runs in ios mode. I didn't want to give the impression that this is _the_ a11y test for this component like some other components have.

Reply:
works for me! fixed in [8b78e5a](https://github.com/ionic-team/ionic-framework/pull/28656/commits/8b78e5a6bafa1ed41907612e6c8071956d291cba).